venue/tasks.py

```diff
@@ -54,110 +62,149 @@ def get_expected_links(code):
+            'forum_profile_id': str(forum_profile.id),
+            'forum_user_id': forum_profile.forum_user_id
+        }
+    }
+    logger.info('Scraping forum profile started', opts)
     if test_mode is None:
         test_mode = config.TEST_MODE
-    if forum_profile.dummy:
-        pass
-    else:
+    if not test_scrape_config:
+        if forum_profile.dummy:
+            return 'dummy'
+    try:
         scraper = load_scraper(forum_profile.forum.scraper_name)
         expected_links = get_expected_links(forum_profile.signature.code)
+        fallback = None
+        # Trigger the use of fallback scraping method after 3 retries
+        if num_retries > 2:
+            fallback = 'crawlera'
+        # Call the scraper
         results = scraper.verify_and_scrape(
             forum_profile.id,
             forum_profile.forum_user_id,
             expected_links,
             vcode=forum_profile.verification_code,
             test_mode=test_mode,
-            test_signature=forum_profile.signature.test_signature)
-        status_code, page_ok, signature_found, total_posts, username, position = results
-        del username
-        # Update forum profile page status
-        status_list = forum_profile.last_page_status
-        if len(status_list):
-            old_status = status_list[-1]
-            new_status_list = [old_status]
+            test_signature=forum_profile.signature.test_signature,
+            fallback=fallback,
+            test_config=test_scrape_config)
+        if test_scrape_config:
+            return results
         else:
-            new_status_list = []
-        new_status = {
-            'status_code': status_code,
-            'page_ok': page_ok,
-            'signature_found': signature_found
-        }
-        new_status_list.append(new_status)
-        forum_profile.last_page_status = new_status_list
-        forum_profile.save()
-        # Update the forum user rank, if it changed
-        forum_rank = ForumUserRank.objects.get(name=position)
-        if forum_profile.forum_rank != forum_rank:
-            forum_profile.forum_rank = forum_rank
-            forum_profile.save()
-        # Get the current last scrape timestamp
-        last_scrape = forum_profile.get_last_scrape()
-
-        # Check posts that haven't reached maturatation
-        tracked_posts = []
-        for post in forum_profile.posts.filter(matured=False):
-            # Check if it's not due to mature yet
-            post_timestamp = post.timestamp.replace(tzinfo=None)
-            tdiff = timezone.now().replace(tzinfo=None) - post_timestamp
-            tdiff_hours = tdiff.total_seconds() / 3600
-            if tdiff_hours > config.MATURATION_PERIOD:
-                post.matured = True
-                post.date_matured = timezone.now()
-                post.save()
+            status_code, page_ok, signature_found, total_posts, username, position, fallback = results
+            del username
```

Comment:
`del varibale` doesn't give you anithig here as the garbage collector will be running in the end of the execution of this function. 
And if you don't want to get `username`, I consider you yo write `_` instead.
`status_code, page_ok, signature_found, total_posts, _, position, fallback = results`